demo/image_demo.py

```diff
@@ -34,9 +33,6 @@ def parse_args():
 
 
 def main(args):
-    # register all modules in mmdet into the registries
-    register_all_modules()
-
```

Comment:
Do we need to call `init_default_scope` here?

Reply:
The default_scope is initialized in inference api.